src/lynx-scoreboard.ts

```diff
@@ -120,6 +131,15 @@ export class LynxScoreboard {
                 this._publish("error", err.message);
             })
             .bind(this.port, this.ip);
+    }
+
+    private startListeningTCP(): void {
+        // Start TCP listener
+        this._serverTCP.listen(this.port, this.ip);
```

Comment:
Use- https://nodejs.org/docs/latest-v14.x/api/net.html#net_net_createserver_options_connectionlistener

Reply:
I think everything was here from the docs, but the comments were off and the order was a bit confusing.  Is this better?